### PR TITLE
Turmite Arena post

### PR DESCRIPTION
**Pull request by komiamiko**
I made a webapp in a day! Simulate many turmites on a large shared canvas. The post is there to accompany the app itself.

You can leave testing of the webapp to me, unless you feel like running a local copy of the site. You can grab the interface and JS and run it in a playground or something though, as an option with less setup.

I just want my reviewers to review the actual post, make sure the writing and all is okay, before it goes up hopefully tomorrow.

### CODE REVIEW

**Comment by komiamiko** [comment]
I'm considering removing the first section, currently titled "Inspiration". I thought it would be an interesting story and harmlessly expose some of the mess that is my side project ideas. It may detract from the post overall though, and it can be said to pollute the turmite explanations with something unrelated. What do you think?

**Comment by komiamiko** [comment]
If I do end up making an AI project, that'll probably be the one I make first. It's tempting to say this is planning for the future, but I could always introduce the story instead when and if that post goes up, rather than now.

File: _posts/2020-06-07-turmite-arena.markdown [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+</div>
+
+# About Turmite Arena
+{: #about-game}
+
+{% include collapser.markdown %}
+
+<div>
+
+## Inspiration
```
Comment (simcard0000) [inline _posts/2020-06-07-turmite-arena.markdown:98]: Speaking about your inspiration in my opinion is a good hook into explaining the actual project 🐜

File: _posts/2020-06-07-turmite-arena.markdown [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+{: #about-inspiration}
+
+{% include collapser.markdown %}
+
+<div>
+
+I always have more potential project ideas bouncing around than I had time for, and putting several turmites in a big arena was not originally one of them.
```
Comment (simcard0000) [inline _posts/2020-06-07-turmite-arena.markdown:105]: I think it should be "I always have more potential project ideas bouncing around than I *have* time for"?

File: _posts/2020-06-07-turmite-arena.markdown [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+
+Originally, my plan was for a first machine learning project with an artistic focus.
```
Comment (simcard0000) [inline _posts/2020-06-07-turmite-arena.markdown:107]: I find this starting sentence a bit confusing. Was the machine learning project going to be the first project you worked on, or are you saying you were going to make *the first* machine learning project with an artistic focus?

File: _posts/2020-06-07-turmite-arena.markdown (hunk 2 above)
Comment (komiamiko) [reply _posts/2020-06-07-turmite-arena.markdown:105]: yep that's mistake, thanks for spotting it

File: _posts/2020-06-07-turmite-arena.markdown (hunk 3 above)
Comment (komiamiko) [reply _posts/2020-06-07-turmite-arena.markdown:107]: my first project

will fix

**Comment by komiamiko** [comment]
It's live! 🎉 Thanks for your help!

I'll still respond to any bug reports or make any minor but highly requested quality of life changes, but otherwise, this is the final state of the post. There is value in having a static archive that you can always come back to, so I won't make any big changes now.

Enjoy Turmite Arena!

**Pull request merged**